unity-renderer/Assets/DCLServices/EmotesCatalog/EmotesCatalogService/WebInterfaceEmotesCatalogService.cs

```diff
@@ -136,6 +135,9 @@ private void EmbedEmotes()
 
         public bool TryGetLoadedEmote(string id, out WearableItem emote) { return emotes.TryGetValue(id, out emote); }
 
+        public UniTask<WearableItem> RequestEmoteFromBuilderAsync(string emoteId, CancellationToken cancellationToken) =>
+            throw new NotImplementedException("Implemented in LambdasEmotesCatalogService");
```

Comment:
I am not sure about this approach (seems like code smell). Basic guideline for such exception is that  _`NotImplementedException` should be a temporary measure. Before releasing software to production, all instances of this exception should be addressed to ensure that the application is complete and fully functional._ ... and .._`NotImplementedException` is used to indicate that a particular method, property, or functionality has not **YET** been implemented_

Maybe we can better separate interfaces to avoid having these exceptions?

Reply:
`WebInterfaceEmotesCatalogService` and `WebInterfaceWearablesCatalogService` are marked as obsolete and could be removed after this implementation. They are only being used to preview wearables & emotes in the backpack v1. This implementation is deprecating that flow. So i think its over-engineering at this point to segregate into new interfaces.